### ISSUE TITLE
Remove the rounding.

### ISSUE THREAD

**Issue by frodberserk** [hoang8f/android-segmented-control#20]
Great library :). Is it possible to remove the rounding?

**Comment by kdani41** [comment]
@frodberserk I guess you probably need to set "radio_button_conner_radius" in dimens to 0.

**Comment by tchar** [comment]
@frodberserk Yes just replace the value of "radio_button_conner_radius" in dimens to whatever you want. When I'm not short on time I'll try to add an xml attribute so you can have multiple types (different radius corners) in the same android app.

**Comment by frodberserk** [comment]
Thanks for the reply! :)

**Comment by kdani41** [comment]
@tchar same sort of question but i couldn't figure out a way to take the outlines off the segmented control. Basically i don't want any outlines in an unselected stated. I tried to comment the following in respective radio_unchecked_left and right 
       
    <stroke>
         android:height="1dp"
         android:color="@color/radio_button_selected_color" />

But i guess in the java file it's setting the outlines again.

**Comment by frodberserk** [comment]
I fixed this by using 2 extra radio buttons with 0dp width, one on the left side and one on the right side :D

**Comment by kdani41** [comment]
@frodberserk Adding 2 extra radio buttons of 0 width in the layout  produces nothing for me :( I still have the outlines.
Are you talking about the strokeWidth ?

**Comment by frodberserk** [comment]
@kdani41 I used like this:

<info.hoang8f.android.segmented.SegmentedGroup
        android:id="@+id/category"
        android:layout_width="match_parent"
        android:layout_gravity="bottom"
        android:background="@color/app_tab_view_profile_shop"
        android:layout_height="wrap_content"
        android:orientation="horizontal"
        >

        <RadioButton
            android:layout_width="0dp"
            android:layout_height="wrap_content"
            style="@style/RadioButtonNew" />

        <RadioButton
            android:id="@+id/looks"
            android:layout_width="0dp"
            android:padding="12dp"
            android:layout_weight="1"
            android:layout_height="wrap_content"
            android:text="Looks"
            style="@style/RadioButtonNew"
            android:checked="true"
           />
        <RadioButton
            android:id="@+id/items"
            android:layout_width="0dp"
            android:padding="12dp"
            android:layout_weight="1"
            android:layout_height="wrap_content"
            android:text="Items"
            style="@style/RadioButtonNew"

             />

        <RadioButton
            android:layout_width="0dp"
            android:layout_height="wrap_content"
            style="@style/RadioButtonNew" />
    </info.hoang8f.android.segmented.SegmentedGroup>

**Comment by tchar** [comment]
@kdani41 I guess if you want not to have outlines on unselected buttons you will have to do it programmatically. Except if you want to set the stroke's width to 0dp for all radio buttons.

**Comment by frodberserk** [comment]
@kdani41 You are right. I used this code only to remove roundings :P

**Comment by kdani41** [comment]
Thanks @frodberserk..
@tchar thanks you're absolutely right. I ended up changing this line in the java file.

    oneDP = (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 0, resources.getDisplayMetrics());

**Comment by tchar** [comment]
@kdani41 Ok i haven't contributed to this project in a while or so and I hope I am not overriding the local library with the one from maven, but it seems I cannot change the stroke width itself.
I added a new dimension and set all the strokes width to that dimension. Changing it does nothing.
Also if I delete the whole stroke element it keeps rendering the buttons the same way.
I am sort on time now because of exams, but when I'm done I'll attend to it.

**Comment by kdani41** [comment]
@tchar yup that's exactly what was happening with me. So i changed this line in java file
from this 
    
    oneDP = (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 1, resources.getDisplayMetrics());
 
to this 

    oneDP = (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 0, resources.getDisplayMetrics());

Thanks for the help :) Good luck with exams :+1:

**Comment by tchar** [comment]
@kdani41 

Ok I will post a fix in a week or so.
Here is the case though and I advice you to do so.
Create a value inside library's dimension file e.g radio_button_stroke_width.
Then assign all occurences of stroke:width to point to that new dimens value.

Also put the same dimens value in your main project (outside the library) this will be used to override the stroke:width within your application (because messing with libraries is not the best option. Maven is an example).

Then change all occurences from oneDp to marginDp and replace
the assignment from the old one to this:

        marginDp = (int) getResources().getDimension(R.dimen.stroke_width);

Alternatively you can define it into the radiogroup's atributes inside the layout xml and parse it from there.

Now you can change the border's dp from your app's dimens.xml without messing with the library.

**Comment by kdani41** [comment]
@tchar thanks you're absolutely correct. This is the best way of doing it.
